Cargo.lock

```diff
@@ -412,9 +412,9 @@ version = "0.1.0"
 
 [[package]]
 name = "cc"
-version = "1.0.57"
+version = "1.0.58"
```

Comment:
FWIW one useful thing I've seen is that you may want to increase the version requirement in `Cargo.toml` of a crate like `std` (or something else that pulls it in in this repo). That way if others end up having an accidental merge conflict on `Cargo.lock` everyone will always be forced to pull in at least this version.

Reply:
These are the Cargo.toml files that pull in both crates:

- src/backtrace/crates/backtrace-sys/Cargo.toml
- src/backtrace/crates/line-tables-only/Cargo.toml
- src/bootstrap/Cargo.toml
- src/librustc_codegen_ssa/Cargo.toml
- src/librustc_llvm/Cargo.toml
- src/libunwind/Cargo.toml

The specific crates that I remember being affected at compile time were `compiler_builtins` and `backtrace-sys`

`backtrace` is a submodule so I shouldn't update that. `bootstrap` feels too confusing. The others are all about the same in my mind, so I'll pick `librustc_llvm`. Please let me know if a different location makes sense.

Reply:
Eh I think w/e is fine. Picking any like `librustc_llvm` should be fine